wp-content/themes/core/assets/css/src/theme/typography/_utilities.pcss

```diff
@@ -139,3 +139,28 @@
 .t-tag {
 	@mixin t-tag;
 }
+
+/* -------------------------------------------------------------------------
+ * Text Alignment
```

Comment:
One additional ask: Can you mention in the comment header here that the `has-*` styles are added by WP Core/Gutenberg? (Future-me will appreciate knowing why we have 2 selectors for the same purpose.)